client.go

```diff
@@ -170,3 +171,92 @@ func Exec(ctx context.Context, cmd string, args ...interface{}) error {
 func Query(ctx context.Context, cmd string, args ...interface{}) Args {
 	return DefaultClient.Query(ctx, cmd, args...)
 }
+
+type MultiError []error
```

Comment:
Thanks :)

I exported it because it's declared in package `redis` but used in package `main`.

Is there a better way to do this ?